doc-content/enterprise-only/employee-rostering/optashift-ER-runningwar-proc.adoc

```diff
@@ -0,0 +1,30 @@
+[id='optashift-ER-runningwar-proc']
+= Running the supplied pre-built WAR file
+
+You can deploy and run the pre-built `employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` file that is supplied with {PRODUCT}.
+
+.Prerequisites
+* You prepared the deployment files as described in <<er-deploy-prepare-proc>>.
+* A Java Development Kit is installed.
+* You have a deployed WildFly or {EAP} application server.
+
+.Procedure
+. In your WildFly or {EAP} application server, enable property replacement. For more information about enabling and disabling property replacement in {EAP}, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/configuration_guide/jboss_eap_management#property_replacement[Property Replacement].
+.. Start the application server in standalone mode.
+.. Start the management CLI by entering the following command:
++
+[source,bash]
+----
+EAP_HOME/bin/jboss-cli.sh --connect
+----
++
+.. In the management CLI enter the following command:
++
+[source]
+----
+/subsystem=ee:write-attribute(name="spec-descriptor-property-replacement",value=VALUE)
+----
++
+. Stop the application server.
+. Deploy the `binaries/employee-rostering-webapp-{MAVEN_ARTIFACT_VERSION}.war` directory into the application server and start the application server.
+. Access the application at http://localhost:8080/gwtui/gwtui.html
```

Comment:
Needs a full stop. But I would reword something like:
To start the application, enter BLAH in a web browser.